src/catalogue/items/catalogueItemsTable.component.test.tsx

```diff
@@ -1,81 +1,154 @@
 import React from 'react';
-import { renderComponentWithBrowserRouter } from '../../setupTests';
-import { screen, waitFor, within } from '@testing-library/react';
+import {
+  getCatalogueCategoryById,
+  renderComponentWithBrowserRouter,
+} from '../../setupTests';
+import { screen, waitFor } from '@testing-library/react';
 import CatalogueItemsTable, {
   CatalogueItemsTableProps,
 } from './catalogueItemsTable.component';
 import userEvent from '@testing-library/user-event';
 
+// jest.setTimeout(20000);
 describe('Catalogue Items Table', () => {
   let props: CatalogueItemsTableProps;
   let user;
-  const onChangeCatalogueItemDetails = jest.fn();
-  const onChangeCatalogueItemManufacturer = jest.fn();
-  const onChangeCatalogueItemPropertyValues = jest.fn();
-  const onChangeAddItemDialogOpen = jest.fn();
 
   const createView = () => {
     return renderComponentWithBrowserRouter(<CatalogueItemsTable {...props} />);
   };
 
+  const hideOrShowColumn = async (columns: string[]) => {
```

Comment:
```suggestion
  const ensureColumnsVisible = async (columns: string[]) => {
```
I initially thought the tests were missing 'expects' due to the naming, perhaps this would imply it also checks they exist afterwards as well?